### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by TemitopeAgbaje**
### Type of Change

- [ #] Adding a new entry
- [# ] Editing an existing entry (fixing a typo, bug, issues, etc)
- [# ] Updating the documentation

### Checklist

- [# ] My entry follows the Codecademy Docs style guide
- [# ] I have performed a self-review of my own writing and code
- [ #] I have made corresponding changes to the documentation
- [# ] My changes generate no new warnings
- [ #] I have checked my entry and corrected any misspellings
- [ #] All writings are my own

I corrected some misspellings and typo

### CODE REVIEW

File: content/html/concepts/semantic-html/terms/figcaption/figcaption.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ For example:
 ```html
 <figure>
   <img src="overwatch.jpg">
-  <figcaption>This picture shows characters from Overwatch.</figcaption>
+  <figcaption>This picture shows characters from overwatch.</figcaption>
```
Comment (sonnynomnom) [inline content/html/concepts/semantic-html/terms/figcaption/figcaption.md:33]: Hi @TemitopeAgbaje, Overwatch is a game. So let's keep this capitalized.

Comment (sonnynomnom) [reply content/html/concepts/semantic-html/terms/figcaption/figcaption.md:33]: Feel free to edit this to say:
 
`<figcaption>This picture shows characters from the video game Overwatch.</figcaption>`

File: content/html/concepts/semantic-html/terms/footer/footer.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ The `<footer>` semantic tag represents a part of a page which is meant to be at
 
 ### Website Footer
 
-After the main content is shown to the users of Bongo Ltd., they should see navigation links to the pages that aren't featured in the header as well as copyright information for the current page.
+After the main content is shown to the users of Bongo Ltd., they should see navigation links to the pages that are not featured in the header as well as copyright information for the current page.
```
Comment (sonnynomnom) [inline content/html/concepts/semantic-html/terms/footer/footer.md:31]: 👍

File: content/html/concepts/semantic-html/terms/figure/figure.md [hunk 3]
```diff
@@ -12,7 +12,7 @@ CatalogContent:
   - "paths/front-end-engineer-career-path"
 ---
 
-The `<figure>` semantic HTML element is used to encapsulate media such as an image, diagram, or code snippet. 
+The `<figure>` semantic HTML element is used to encapsulate media such as an image, diagram or code snippet.
```
Comment (sonnynomnom) [inline content/html/concepts/semantic-html/terms/figure/figure.md:15]: Let's keep the Oxford comma here.

**Review by sonnynomnom — CHANGES_REQUESTED** [review]
Hi @TemitopeAgbaje, make sure to sign the CLA and make the changes. And I can merge!

File: content/html/concepts/semantic-html/terms/figcaption/figcaption.md (hunk 1 above)
Comment (TemitopeAgbaje) [reply content/html/concepts/semantic-html/terms/figcaption/figcaption.md:33]: Okat @sonnynomnom  would correct that back

File: content/html/concepts/semantic-html/terms/figure/figure.md (hunk 3 above)
Comment (TemitopeAgbaje) [reply content/html/concepts/semantic-html/terms/figure/figure.md:15]: okay

**Comment by TemitopeAgbaje** [comment]
i pushed the correction but i can not make a pr

**Comment by TemitopeAgbaje** [comment]
I already pushed the corrections
kindly check

**Review by sonnynomnom — COMMENTED** [review]
LGTM

**Review by sonnynomnom — APPROVED** [review]

**Comment by sonnynomnom** [comment]
Just realized there was an extra space. Deleted. 

We are good!

**Pull request merged**